### ISSUE TITLE
Hawkular-metrics 3.6.0-alpha.0 does not start

### ISSUE THREAD

**Issue by gordillo-ramon-redhat** [openshift/origin#13533]
```
2017-03-16 23:25:19,093 ERROR [stderr] (default task-13) Exception in thread "default task-13" java.lang.NoSuchMethodError: sun.security.ssl.SupportedEllipticCurvesExtension.curveIds()[I
2017-03-16 23:25:19,109 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.setupEphemeralECDHKeys(ServerHandshaker.java:1470)
2017-03-16 23:25:19,109 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.trySetCipherSuite(ServerHandshaker.java:1272)
2017-03-16 23:25:19,109 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.chooseCipherSuite(ServerHandshaker.java:1076)
2017-03-16 23:25:19,109 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.clientHello(ServerHandshaker.java:742)
2017-03-16 23:25:19,110 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.processMessage(ServerHandshaker.java:225)
2017-03-16 23:25:19,110 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
2017-03-16 23:25:19,110 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker$1.run(Handshaker.java:966)
2017-03-16 23:25:19,111 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker$1.run(Handshaker.java:963)
2017-03-16 23:25:19,111 ERROR [stderr] (default task-13) 	at java.security.AccessController.doPrivileged(Native Method)
2017-03-16 23:25:19,137 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker$DelegatedTask.run(Handshaker.java:1416)
2017-03-16 23:25:19,137 ERROR [stderr] (default task-13) 	at io.undertow.protocols.ssl.SslConduit$5.run(SslConduit.java:1023)
2017-03-16 23:25:19,138 ERROR [stderr] (default task-13) 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
2017-03-16 23:25:19,138 ERROR [stderr] (default task-13) 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
2017-03-16 23:25:19,138 ERROR [stderr] (default task-13) 	at java.lang.Thread.run(Thread.java:745)
```

##### Version
oc v3.6.0-alpha.0+0343989
kubernetes v1.5.2+43a9be4
features: Basic-Auth GSSAPI Kerberos SPNEGO

Server https://172.17.0.1:8443
openshift v3.6.0-alpha.0+0343989
kubernetes v1.5.2+43a9be4
 
##### Steps To Reproduce
It has been deployed manually, due to some problems with oc-cluster up --metrics:

oc process -f https://raw.githubusercontent.com/openshift/openshift-ansible/master/roles/openshift_hosted_templates/files/v1.5/origin/metrics-deployer.yaml \
	-p IMAGE_PREFIX=openshift/origin- \
	-p IMAGE_VERSION=v3.6.0-alpha.0 \
        -p HAWKULAR_METRICS_HOSTNAME=metrics-openshift-infra.apps.172.17.0.1.nip.io \
	-p USE_PERSISTENT_STORAGE=false \
        -p MODE=deploy \
        -p METRIC_RESOLUTION=10s | oc create -n openshift-infra

##### Additional Information
Seems to be something related with OpenJDK 1.8.0_121

https://issues.jboss.org/browse/HWKMETRICS-632

@jorgemoralespou

**Comment by isnuryusuf** [comment]
got same problem on origin 1.5.0

2017-03-23 23:46:38,294 ERROR [stderr] (default task-13) Exception in thread "default task-13" java.lang.NoSuchMethodError: sun.security.ssl.SupportedEllipticCurvesExtension.curveIds()[I
2017-03-23 23:46:38,295 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.setupEphemeralECDHKeys(ServerHandshaker.java:1470)
2017-03-23 23:46:38,295 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.trySetCipherSuite(ServerHandshaker.java:1272)
2017-03-23 23:46:38,295 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.chooseCipherSuite(ServerHandshaker.java:1076)
2017-03-23 23:46:38,295 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.clientHello(ServerHandshaker.java:742)
2017-03-23 23:46:38,295 ERROR [stderr] (default task-13) 	at sun.security.ssl.ServerHandshaker.processMessage(ServerHandshaker.java:225)
2017-03-23 23:46:38,296 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker.processLoop(Handshaker.java:1026)
2017-03-23 23:46:38,296 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker$1.run(Handshaker.java:966)
2017-03-23 23:46:38,296 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker$1.run(Handshaker.java:963)
2017-03-23 23:46:38,296 ERROR [stderr] (default task-13) 	at java.security.AccessController.doPrivileged(Native Method)
2017-03-23 23:46:38,296 ERROR [stderr] (default task-13) 	at sun.security.ssl.Handshaker$DelegatedTask.run(Handshaker.java:1416)
2017-03-23 23:46:38,296 ERROR [stderr] (default task-13) 	at io.undertow.protocols.ssl.SslConduit$5.run(SslConduit.java:1023)
2017-03-23 23:46:38,296 ERROR [stderr] (default task-13) 	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
2017-03-23 23:46:38,297 ERROR [stderr] (default task-13) 	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
2017-03-23 23:46:38,297 ERROR [stderr] (default task-13) 	at java.lang.Thread.run(Thread.java:745)
2017-03-23 23:46:41,089 ERROR [stderr] (default task-15) Exception in thread "default task-15" java.lang.NoSuchMethodError: sun.security.ssl.SupportedEllipticCurvesExtension.curveIds()[I
2017-03-23 23:46:41,089 ERROR [stderr] (default task-15) 	at sun.security.ssl.ServerHandshaker.setupEphemeralECDHKeys(ServerHandshaker.java:1470)

**Comment by mwringe** [comment]
There is an issue with 3.6.0-alpha.0. It been fixed in the latest images, but we still need to figure out how to push out updated images for 3.6.0-alpha.0

But there should be no problem with 3.5. What command are you using to deploy the 3.5 version?

**Comment by gordillo-ramon-redhat** [comment]
Fixed for me in 3.6.0-alpha.0 with the updated image.
